docs/userguide/intro.md

```diff
@@ -64,6 +65,24 @@ learning how to manage data, volumes and mounts inside our containers.
 
 Go to [Managing Data in Containers](../tutorials/dockervolumes.md).
 
+## Managing Metadata (Labels) for Docker Objects
+
+_Labels_ are a mechanism for applying metadata to Docker objects, including:
+
+- Images
+- Containers
+- Daemons
```

Comment:
I'd use `Local daemon`. There is no concept of multiple daemons, then they are already swarm nodes.